lib/livebook_web/components/core_components.ex

```diff
@@ -803,13 +797,13 @@ defmodule LivebookWeb.CoreComponents do
             >
               <div class="block p-4 sm:px-6">
                 <span class="absolute -inset-y-px right-0 -left-4 group-hover:bg-gray-100 sm:rounded-l-xl" />
-                <span class={["relative", i == 0 && "text-sm font-medium text-gray-900"]}>
+                <span class={["relative", i == 0]}>
```

Comment:
```suggestion
                <span class="relative">
```

Reply:
Since you are no longer using `i`, you can replace `:for={{col, i} <- Enum.with_index(@col)}` a couple lines above by `:for={col <- @col}`.